ui/topSection/components/defenseBlock.pug

```diff
@@ -1,24 +1,23 @@
 // Defense 
 .defense-block
   .defenses.epic-table-header-grid
-    div
-    div Melee
-    div Range
-  .defenses-table(data-epic-table-name="defenses")
+    div Def.
+    div Ability
+    div.armor-header.epic-tooltip Armor
+      span.epic-tooltip-text.right Ability without Armor
+  .defenses-table-row.bonus-row(data-epic-table-name="defenses")
     .table-label +bonus
-    input.center.bold.larger(name='attr_melee_boost' type='number' value='0')
-    input.center.bold.larger(name='attr_ranged_boost' type='number' value='0')
+    input.center.bold.larger(name='attr_defense_boost' type='number' value='0')
   .defenses-table-row(data-epic-table-name="defenses")
     .table-label.larger Dodge
     input(name='attr_dodge' type='hidden')
-    span.center.bold.larger(name='attr_current_dodge_melee' value='0')
-    span.center.bold.larger(name='attr_current_dodge_ranged' value='0')
-  .defenses-table-row(data-epic-table-name="defenses")
-    .table-label.larger Parry
-    input.center.bold.larger(name='attr_current_parry_melee' type='number' value='0')
-    span.center.bold.larger --
+    span.center.bold.larger(name='attr_current_dodge_with_armor' value='0')
+    span.center.bold.larger(name='attr_current_dodge_without_armor' value='0')
   .defenses-table-row(data-epic-table-name="defenses")
     .table-label.larger Block
     input(name='attr_block' type='hidden')
-    span.center.bold.larger(name='attr_current_block_melee' value='0')
-    span.center.bold.larger(name='attr_current_block_ranged' value='0')
+    span.center.bold.larger(name='attr_current_block_with_armor' value='0')
+    span.center.bold.larger(name='attr_current_block_without_armor' value='0')
+  .defenses-table-row.parry-row(data-epic-table-name="defenses")
+    .table-label.larger Parry
+    input.center.bold.larger(name='attr_current_parry' type='number' value='0')
```

Comment:
I think we should follow up with a separate ticket to move parry to be a column in Weapons as mentioned, and rethink this row.